src/main/java/org/aerogear/kryptowire/KryptowireServiceImpl.java

```diff
@@ -19,15 +22,16 @@
     private static final Logger logger = Logger.getLogger(KryptowireServiceImpl.class.getName());
 
     private String apiKey;
-
     private String apiEndpoint;
 
-    public KryptowireServiceImpl(String apiEndpoint, String apiKey) {
-        if (this.apiEndpoint != null && !this.apiEndpoint.isEmpty() && !this.apiEndpoint.endsWith("/")) {
-            this.apiEndpoint += "/";
+    public KryptowireServiceImpl(String apiEndpointParam, String apiKeyParam) {
```

Comment:
Any reason you added param to the end of these?

Reply:
`mvn install` was complaining about those values being the same